spec/system/prior_authority/view_applications_spec.rb

```diff
@@ -43,5 +45,7 @@
     click_on 'LAA-1234'
     expect(page).to have_current_path prior_authority_application_path(application)
     expect(page).to have_content 'Requested: £80.50'
+    click_on 'test.pdf'
```

Comment:
should this test not be more extensively testing the expected content of the render "cards" if we are to rely on system tests more than unit tests. coverage will be green but are we covering sufficient permutations and expected content?

Reply:
Yup, will add some more specific expectations today. This ticket is now in QA so I expect there'll be some back and forth before it's merged, which gives me plenty of time to improve the tests.